in_toto/runlib.py

```diff
@@ -214,9 +219,21 @@ def record_artifacts_as_dict(artifacts, exclude_patterns=None,
       # *nix filepaths. A better solution may be in order though...
       artifact = artifact.replace('\\', '/')
 
-      # Path was already normalized above
-      artifacts_dict[artifact] = _hash_artifact(artifact,
-          normalize_line_endings=normalize_line_endings)
+      if lstrip_paths:
+        # If a prefix is passed using the argument --lstrip-paths,
+        # that prefix is left stripped from the filepath passed.
+        # Currently, it's set to only take a single prefix in common_args.
+        # Note: if the prefix doesn't include a trailing /, the dictionary key
+        # may include an unexpected /.
+        # Path was already normalized above
+        if artifact.startswith(lstrip_paths[0]):
+          stripped_artifact = artifact[len(lstrip_paths[0]):]
+        artifacts_dict[stripped_artifact] = _hash_artifact(artifact,
+            normalize_line_endings=normalize_line_endings)
+      else:
+        # Path was already normalized above
+        artifacts_dict[artifact] = _hash_artifact(artifact,
+            normalize_line_endings=normalize_line_endings)
```

Comment:
It feels that this else clause is repetitive, is there a reason we can't move 231-232 out of the if and ignore the else?